tests/scripts/format.py

```diff
@@ -85,13 +85,13 @@ def main(argv):
 
     print("Downloading .Net CLI")
     if platform == 'Linux':
-        dotnetcliUrl = "https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0/dotnet-sdk-2.0.0-linux-x64.tar.gz"
+        dotnetcliUrl = "https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.300/dotnet-sdk-2.1.300-linux-x64.tar.gz"
         dotnetcliFilename = os.path.join(dotnetcliPath, 'dotnetcli-jitutils.tar.gz')
     elif platform == 'OSX':
-        dotnetcliUrl = "https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0/dotnet-sdk-2.0.0-macos-x64.tar.gz"
+        dotnetcliUrl = "https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.300/dotnet-sdk-2.1.300-macos-x64.tar.gz"
```

Comment:
This returns 404. I think the correct URLs are of 2.1 RTM https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.1/dotnet-runtime-latest-osx-x64.tar.gz and the rest can be found here: https://github.com/dotnet/core-setup#daily-builds

Reply:
Thanks. Offhand, do you know if the old URL worked?

I would prefer using the actual RTM build here instead of a nightly, even if it should work the same. So I'll have to figure out where the RTM builds ended up.

The CI currently doesn't use jitutils on OSX legs. But it would be nice to get it right.

Reply:
Old URL isn't working either. The correct 2.0 URL for SDK on Mac is:
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/2.0.0/dotnet-sdk-latest-osx-x64.tar.gz
2.1 URL for SDK on Mac:
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/2.1.3xx/dotnet-sdk-latest-osx-x64.tar.gz

Those are listed here: https://github.com/dotnet/cli/tree/release/2.1.3xx#installers-and-binaries

Reply:
Replace `https://dotnetcli.blob.core.windows.net/` with CDN `https://dotnetcli.azureedge.net/` :)